CMakeLists.txt

```diff
@@ -2040,14 +2133,14 @@ add_executable(netdata
 )
 
 target_compile_definitions(netdata PRIVATE
-        "$<$<BOOL:${ENABLE_PROTOBUF}>:${PROTOBUF_CFLAGS_OTHER}>"
+        "$<$<BOOL:${ENABLE_PROTOBUF}>:${NETDATA_PROTOBUF_CFLAGS_OTHER}>"
```

Comment:
With/Without bundled protobuf:

```
[841/888] /usr/local/bin/cc -DDLIB_NO_GUI_SUPPORT -DNETDATA_PROTOBUF_CFLAGS_OTHER-NOTFOUND -D_GNU_SOURCE -I/home/vk/repos/nd/pb/build/_deps/protobuf-src/src -I/home/vk/repos/nd/pb/src/aclk/aclk-schemas -I/usr/include/libmongoc-1.0 -I/usr/include/libbson-1.0 -I/usr/include/uuid -I/usr/include/json-c -I/home/vk/repos/nd/pb/build -I/home/vk/repos/nd/pb/src -I/home/vk/repos/nd/pb/src/libnetdata/libjudy/src -I/home/vk/repos/nd/pb/src/libnetdata/libjudy/src/JudyCommon -I/home/vk/repos/nd/pb/aclk/helpers -I/home/vk/repos/nd/pb/src/web/server/h2o/libh2o/include -I/home/vk/repos/nd/pb/build/_deps/absl-src -I/home/vk/repos/nd/pb/src/web/server/h2o/libh2o/deps/cloexec -I/home/vk/repos/nd/pb/src/web/s
erver/h2o/libh2o/deps/brotli/enc -I/home/vk/repos/nd/pb/src/web/server/h2o/libh2o/deps/golombset -I/home/vk/repos/nd/pb/src/web/server/h2o/libh2o/deps/libgkc -I/home/vk/repos/nd/pb/src/web/server/h2o/libh2o/deps/libyrmcds -I/home/vk/repos/nd/pb/src/web/server/h2o/libh2o/deps/klib -I/home/vk/repos/nd/pb/src/web/server/h2o/libh2o/deps/neverbleed -I/home/vk/repos/nd/pb/src/web/server/h2o/libh2o/deps/picohttpparser -I/home/vk/repos/nd/pb/src/web/server/h2o/libh2o/deps/picotest -I/home/vk/repos/nd/pb/src/web/server/h2o/libh2o/deps/yaml/include -I/home/vk/repos/nd/pb/src/web/server/h2o/libh2o/deps/yoml -Wall -Wextra -DNETDATA_LOG_STREAM_RECEIVE -fsanitize=fuzzer-no-link -fexceptions -g   -ffunct
ion-sections -fdata-sections -DMQTT_WSS_CUSTOM_ALLOC -DRBUF_CUSTOM_MALLOC -DMQTT_WSS_CPUSTATS -DH2O_USE_LIBUV=0 -std=gnu11 -MD -MT CMakeFiles/netdata.dir/src/collectors/proc.plugin/proc_uptime.c.o -MF CMakeFiles/netdata.dir/src/collectors/proc.plugin/proc_uptime.c.o.d -o CMakeFiles/netdata.dir/src/collectors/proc.plugin/proc_uptime.c.o -c /home/vk/repos/nd/pb/src/collectors/proc.plugin/proc_uptime.c
In file included from <built-in>:390:
<command line>:2:38: warning: ISO C99 requires whitespace after the macro name [-Wc99-extensions]
    2 | #define NETDATA_PROTOBUF_CFLAGS_OTHER-NOTFOUND 1
      |                                      ^
1 warning generated.
```

Reply:
This issue remains even after the updates.

Reply:
The latest commit should fix this now.